src/containers/Application/Part3.js

```diff
@@ -1,6 +1,166 @@
-import React from 'react'
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Dropdown, Select, TextInput } from '../../components/Input'
+import { H1, QuestionHeading } from '../../components/Typography'
+
+const StyledDropdown = styled(Dropdown)`
+  margin: 1em 0;
+`
```

Comment:
Someone please help I don't know why this style isn't overriding the component styles 😭

Reply:
In hindsight I should have built the dropdown from scratch :(((( I used a library to make the dropdown component, so to override the styling you will need to do a bit more work! 

To modify the styling of react-select , you will need to change the styling with the div classes. Since I think you want to change the margin of the dropdown's select container, you would do something like this inside `StyledDropdown`:

`const StyledDropdown = styled(Dropdown)`
`.react-select__control {`
    `margin: 1em 0;`
`}`

The `Control` component of the dropdown component is the div surrounding the entire select container. If you want to change something about the menu, you would use `.react-select__menu`. You can see the use cases in `/src/components/Input/Dropdown.js`

lmk if u have anymore questions!

Reply:
Ohhh gotcha! Thanks so much! 🙏

Reply:
@andykwan1124 just tried this out in 8393833 but doesn't seem to work :/ any idea where I could be going wrong?

Reply:
ahhhhh no that's mb sorry. I didn't pass in the className component to the dropdown so it couldn't handle custom styling our own components. I will make a PR for this fix really soon, along with a sample use case in the charcuterie page. Look for 'Custom styled dropdown' on it.

Reply:
Oh wait maybe you could try 
```
const StyledDropdown = styled(Dropdown)`
&& {
  margin: 1em 0;
  .react-select__control {
    margin: 0;
  }
}
```

Reply:
I merged with master and it figured it out somehow :o Andy's first suggestion worked! Thanks everyone <3